src/box/lua/config/init.lua

```diff
@@ -578,6 +578,27 @@ function methods.instances(self)
     return res
 end
 
+function methods.instance_uri(self, uri_type, opts)
+    selfcheck(self, 'instance_uri')
+    uri_type = uri_type == nil and 'peer' or uri_type
+    if uri_type ~= 'peer' and uri_type ~= 'sharding' then
+        error(('Expected "peer" or "sharding", got %s'):format(uri_type), 0)
+    end
+    opts = opts == nil and {} or opts
+    if type(opts) ~= 'table' then
+        error(('Expected table, got %s'):format(opts), 0)
+    end
+    if opts.instance ~= nil and type(opts.instance) ~= 'string' then
+        error(('Expected string, got %s'):format(opts.instance), 0)
+    end
```

Comment:
I guess that `type(opts)` and `type(opts.instance)` should be in the `:format()` arguments.

Reply:
Fixed.